datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1873,6 +1874,11 @@ impl LogicalPlan {
                             .as_ref()
                             .map(|expr| format!(" Filter: {expr}"))
                             .unwrap_or_else(|| "".to_string());
+                        let join_type = if filter.is_none() && keys.is_empty() && matches!(join_type, JoinType::Inner) {
+                            "Cross".to_string()
```

Comment:
should we extend `JoinType` enum to support Cross?

Reply:
I think it’s better not to, otherwise it will be similar to having `LogicalPlan::CrossJoin` (I.e. unnecesary).